doc/source/examples/JobAnalysis/job_analysis.ipynb.rst

```diff
@@ -124,94 +124,102 @@ Jobs can be loaded by passing job objects directly, or loading from a path.
 
    from scm.plams import JobAnalysis
 
+::
```

Comment:
Why did this change?

Reply:
The PLAMS output when running jobs appears under whichever cell is being run. Here I think it is confusing so have removed, good spot.